File: java_spring_hw/src/main/java/com/example/java_spring_hw/entity/Task.java
package com.example.java_spring_hw.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.*;
import org.springframework.data.annotation.CreatedDate;

import java.util.Date;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Setter
@Getter
@Table(name = "tasks")
@Entity
public class Task {

    @Id
    @GeneratedValue
    private Long id;

    @CreatedDate
    private Date createdAt;

    private boolean isStarted;

    private String dsc;
}

File: kotlin_quarkus_hw/src/main/kotlin/com/example/entity/Task.kt
package com.example.entity

import java.util.*
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id
import javax.persistence.Table

@Table(name = "tasks")
@Entity
data class Task(
    @Id
    @GeneratedValue
    val id: Long? = null,

    val createdAt: Date? = null,

    val isStarted: Boolean = false,

    val dsc: String? = null
)